scalafmt-tests/src/test/resources/test/arityThreshold.source

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+verticalMultilineAtDefinitionSite = true
+verticalMultilineAtDefinitionSiteArityThreshold = 2
+danglingParentheses = true
+continuationIndent.defnSite = 2
+<<< ONLY Work with continuationIndent.defnSite = 2
+case class Foo(x: String)
+case class Bar(x: String, y: String)
+case class VeryLongNames(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx: String)
+>>>
+case class Foo(x: String)
+case class Bar(
+  x: String,
+  y: String)
+case class VeryLongNames(
+  xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx: String)
```

Comment:
Dangling parentheses is explicitly ignored for `Ctor.Primary`, `Defn.Class`, and `Trait`. I'd prefer to have them dangling in these cases but that seems like a separate issue :)

Reply:
Hmm, is this a peculiarity of `verticalMultilineAtDefinitionSite`? If so I have no opinions, but @pjrt might!

Reply:
@mads-hartmann Yeah, the feature ignores dangling (and most other options). It is one of those features that don't make sense if it listens to every other scalafmt option. It is a holistic style, instead of a piece-meal one (like dangling or align).

As for why, well it looks ugly if you allow dangling. So it only dangles when the class `extends` something.